pipeline-runner/NAMESPACE

```diff
@@ -2,6 +2,7 @@
 
 export(create_seurat)
 export(download_user_files)
+export(extract_features_types)
```

Comment:
I would change features_types to feature_types. Let's not get a new cells_id 🙏

Reply:
sounds good!